model/time.go

```diff
@@ -163,9 +163,20 @@ func (t *Time) UnmarshalJSON(b []byte) error {
 // This type should not propagate beyond the scope of input/output processing.
 type Duration time.Duration
 
+// Set implements pflag/flag.Value
+func (d *Duration) Set(s string) (err error) {
```

Comment:
Umm, so it would make it an extra step as one cannot do `*d, err := ` with the receiver `*d`. 

I can change, but isn't naming the return value a good practice?